src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -129,6 +129,20 @@
     //},
     //"OrchardCore_HealthChecks": {
     //  "Url": "/health/live"
-    //}
+    //},
+    
+    //"OrchardCore_Media_Amazon_S3": {
```

Comment:
Add a docs page for this like https://docs.orchardcore.net/en/latest/docs/reference/modules/Media.Azure/, [see contribution notes](https://docs.orchardcore.net/en/latest/docs/topics/docs-contributions/). Link it here like the others are linked.